bcgov/HOUS-permit-portal
# updating computed compliance capabilities

## Description

Automated compliance messages show up with text (PID, Plan Number, etc.)

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

(https://hous-bssb.atlassian.net/browse/BPHH-653)

## Steps to QA

Create a new application, it should set up PID and plan Number default values and show what the auto-compliance ofund as ad efault value.

## UI Accessibility Checklist


## General Checklist

- [ x] ✅ Provide tests for your changes where relevant
- [ x] 📝 Use descriptive commit messages
- [ x] 📗 Update any related documentation and include any relevant screenshots
